scripts/azure-pipelines/azure-pipelines.yml

```
@@ -59,6 +60,14 @@ jobs:
     vcpkgToolSha: ${{ parameters.vcpkgToolSha }}
     tripletPattern: ${{ parameters.tripletPattern }}
 
+#- template: windows/azure-pipelines.yml

```

```suggestion
- template: windows/azure-pipelines.yml
```
I think here should be no `#`